types/json-rpc-ws/json-rpc-ws-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as JsonRpcWs from 'json-rpc-ws';
+
+let server = JsonRpcWs.createServer();
```

Comment:
This could use an example of providing a generic type argument to `createServer` -- when is that useful?